balboa-http/src/it/scala/com/socrata/balboa/server/load/PostLoadTest.scala

```diff
@@ -0,0 +1,44 @@
+package com.socrata.balboa.server.load
+
+import com.socrata.balboa.metrics.Metric.RecordType
+import com.socrata.balboa.server.{Config, EntityJSON, MetricJSON, ServiceUtils}
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ScenarioBuilder
+import io.gatling.http.Predef._
+import org.json4s.{DefaultFormats, Extraction, Formats}
+import org.json4s.jackson.JsonMethods.{pretty, render}
+
+import scala.concurrent.duration._
+
+class PostLoadTest extends Simulation {
```

Comment:
A quick description here of the intent of these tests. Just something like testing the balboa-http write path by adding <kinds of> metrics.

Since Gatlin is not widely used in our company, the documentation bar is a little higher. People will not necessarily arrive here with a rough general knowledge in place.